tests/core/interfaces/outbound/dataops/test_dataops.py

```diff
@@ -430,6 +431,126 @@ def test_validate(self, config, data, expected_output):
 
         adapter.cleanup()
 
+    @pytest.mark.parametrize(
+        "config, data, expected_output",
+        [
+            # data type error
+            (
+                ValidationConfig(
+                    name="data type error",
+                    columns=[
+                        ColumnValidation(
+                            unique_name="col1",
+                            data_type="not a type",
+                            required=True,
+                            nullable=False,
+                            validations=[],
+                        )
+                    ],
+                    identifying_column_names=["col1"],
+                    validations=[],
+                ),
+                {
+                    "col1": [3, 1],
+                    "col2": [1, 2],
+                },
+                {
+                    "len_unexpected": 2,
+                    "sources": {0: "pydantic/main.py:214:__init__", 1: "dataguard/validator/validator.py:198:validate"},
```

Comment:
Are the integers in there pointing to code lines? If so, then this seems like a maintenance burden?

Reply:
This is a tricky one. My intuition tells me that in a test I would not expect any unexpected errors. But I know that you're going to tell me that there is no other way you can capture these data type conversion errors.